### PR TITLE
Update iOS Common Build Errors.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.asana.com/0/353548986073982/410656625699904

Includes some wording tweaks, and some updated screenshots.

### CODE REVIEW

File: troubleshooting/ios/common_build_errors.adoc [hunk 1]
```diff
@@ -6,46 +6,60 @@ common causes for build failures.
...
-* Rebuild without cache
-* Clone your repo and check if your app builds locally
-* Share your schemes
-* Ensure your tests pass
-* Double check your third party dependencies
+- <<rebuild>>
+- <<clone>>
+- <<schemes>>
+- <<tests>>
+- <<dependencies>>
```
Comment (sshyvr) [inline troubleshooting/ios/common_build_errors.adoc]: Just curious what does this `<<>>` syntax mean?

Comment (bbewan) [inline troubleshooting/ios/common_build_errors.adoc]: Thats's the AsciiDoctor cross-reference syntax: http://asciidoctor.org/docs/user-manual/#internal-cross-references

Using it, the mini-TOC at the top of the page is kept in sync with section title edits.

**Comment by bbdocs** [comment]
See the website here: http://30-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://30-bb-docs.surge.sh

**Pull request merged**